.circleci/config.yml

```diff
@@ -1490,8 +1493,12 @@ jobs:
           destination: builds-mv2
       - store_artifacts:
           path: builds-test
+      - store_artifacts:
+          path: builds-test-mv2
       - store_artifacts:
           path: builds-test-flask
+      - store_artifacts:
+          path: builds-test-flask-mv2
```

Comment:
This was the main issue with the test MV2 builds: we forgot toe `store_artifacts` step.